sdk/requests/logout.js

```diff
@@ -0,0 +1,39 @@
+import { Linking } from 'react-native';
+import { getParameters, setParameters } from '../configuration';
+import { logoutEndpoint } from './endpoints';
+
+const logout = async () => {
+  const parameters = getParameters();
+  let resolveFunction;
+  let rejectFunction;
+  const promise = new Promise((resolve, reject) => {
+    resolveFunction = resolve;
+    rejectFunction = reject;
+  });
+
+  const handleOpenUrl = event => {
+    const urlCheck = event.url;
+    if (urlCheck) {
+      setParameters({ code: 'empty' });
+      resolveFunction(urlCheck);
+    } else rejectFunction(Error('Invalid post logout redirect uri'));
+    Linking.removeEventListener('url', handleOpenUrl);
+  };
+
+  try {
+    Linking.addEventListener('url', handleOpenUrl);
+    if (parameters.idToken && parameters.postLogoutRedirectUri)
+      await Linking.openURL(logoutEndpoint());
+    else {
+      Linking.removeEventListener('url', handleOpenUrl);
+      rejectFunction(Error("Couldn't make request"));
```

Comment:
```suggestion
      rejectFunction(Error("Didn't provide idToken nor postLogoutRedirectUri"));
```
Me parece que queda mucho más explícito el error. Yo le voy a cambiar eso para que el login lo tenga.

Reply:
Creo que el 'nor' debería ser un 'or' :) Sino algo tipo Mising required parameter: idToken / missingRequiredParameter: postLogoutRedirectUri (no sé si podemos distinguir cual falta, si no ponerlos ambos)